diff --git a/src/Backends/DRMBackend.cpp b/src/Backends/DRMBackend.cpp
index d7f935d..d1cf43e 100644
--- a/src/Backends/DRMBackend.cpp
+++ b/src/Backends/DRMBackend.cpp
@@ -562,9 +562,11 @@ static void drm_unset_connector( struct drm_t *drm );
 
 static constexpr uint32_t s_kSteamDeckLCDRates[] =
 {
+	30, 31, 32, 33, 34, 35, 36, 37, 38, 39,
 	40, 41, 42, 43, 44, 45, 46, 47, 48, 49,
 	50, 51, 52, 53, 54, 55, 56, 57, 58, 59,
-	60,
+	60, 61, 62, 63, 64, 65, 66, 67, 68, 69,
+	70,
 };
 
 static constexpr uint32_t s_kSteamDeckOLEDRates[] =
